api/api.proto

```diff
@@ -647,45 +648,95 @@ message StorageCluster {
 service OpenStorageCluster {
   // Enumerate lists all the nodes in the cluster.
   rpc Enumerate(ClusterEnumerateRequest)
-    returns (ClusterEnumerateResponse) {}
+    returns (ClusterEnumerateResponse) {
+      option(google.api.http) = {
```

Comment:
i am curious, this option is part of proto spec as in protoc will take care of generating required code or requires specific plugins other than grpc plugin for go?